bedrock/firefox/views.py

```diff
@@ -744,7 +745,17 @@ class FirefoxHubView(BlogPostsView):
     blog_posts_template_variable = 'articles'
     blog_slugs = 'firefox'
     blog_tags = ['home']
-    template_name = 'firefox/hub/home.html'
+
+    def get_template_names(self):
+        locale = l10n_utils.get_locale(self.request)
+
+        if switch('firefox-57-release') and lang_file_is_active(
+                'firefox/firefox-quantum.lang', locale):
```

Comment:
Appologies I missed this before, it should be `firefox/firefox-quantum`? (no `.lang`)

Reply:
🐍